projects/ngneat/helipopper/src/lib/tippy.directive.ts

```diff
@@ -189,6 +215,35 @@ export class TippyDirective implements OnChanges, AfterViewInit, OnDestroy, OnIn
     this.viewRef = null;
   }
 
+  /**
+   * This method is useful when you append to an element that you might remove from the DOM.
+   * In such cases we want to hide the tooltip and let it go through the destroy lifecycle.
+   */
+  observeHostVisibility() {
+    if (this.props.appendTo) {
+      this.visibilityObserverCleanup?.();
+      this.visibleInternal
```

Comment:
return the subscription?

Reply:
What's the use case that you think the consumer might need it?

Reply:
Unsubscribe sometime else